modules/overrides/userconfigurable/client/limits.go

```diff
@@ -52,6 +55,13 @@ func (l *LimitsMetricsGenerator) GetProcessor() *LimitsMetricsGeneratorProcessor
 	return nil
 }
 
+func (l *LimitsMetricsGenerator) GetCollectionInterval() (time.Duration, bool) {
+	if l != nil && l.CollectionInterval != nil {
+		return *l.CollectionInterval, true
+	}
+	return time.Minute, false
```

Comment:
nit: as this value gets thrown out, it probably makes more sense to use `time.Duration(0)` here. Thoughts?
```suggestion
	return time.Duration(0), false
```

Reply:
I think we can also just return
```suggestion
	return 0, false
```
Doesn't really matter tbh, people shouldn't be reading this value anyway.

Reply:
sounds good!